app/main.py

```diff
@@ -1,67 +1,212 @@
+import hashlib
 import json
+from datetime import datetime, timedelta, timezone
+from functools import lru_cache
+from pathlib import Path
 from typing import MutableMapping
 from uuid import UUID
 
 import uvicorn
 import yaml
 from fastapi import FastAPI, Query, HTTPException
+from pydantic import BaseModel
 from sdc.crypto.jwe_helper import JWEHelper
 from sdc.crypto.key_store import KeyStore
 
-KEY_PURPOSE_SDS = "supplementary_data"
-
 app = FastAPI()
 
 with open("dev-keys.yml", encoding="UTF-8") as keys_file:
     keys = KeyStore(yaml.safe_load(keys_file))
 
+KEY_PURPOSE_SDS = "supplementary_data"
+MOCK_DATA_ROOT_PATH = Path(__file__).parent.parent / "mock_data"
+
+# period_id to match with Launcher
+PERIOD_ID = "201605"
+
+# Currently only have a single schema version
+SCHEMA_VERSION = "v1.0.0"
+
+# Currently only have 1 reporting unit example for each schema version
+TOTAL_REPORTING_UNITS = 1
+
+FORM_TYPES = ["001"]
+
+# Hardcoded paths for now - update if any changes are made to https://github.com/ONSdigital/sds-schema-definitions/tree/main/examples
+MOCK_DATA_PATHS_BY_SURVEY_ID = {
+    "test": ["123"],
+    "prodcom": ["014"],
```

Comment:
I guess worth noting that prodcom and bres/brs cannot be tested yet as there are no schemas for these atm

Reply:
Updated the PR description to explain, but for clarity, should I also pop this in as an inline comment too?